backends/events/ps3sdl/ps3sdl-events.cpp

```diff
@@ -37,9 +37,10 @@
  */
 void PS3SdlEventSource::preprocessEvents(SDL_Event *event) {
 	if (event->type == SDL_APP_DIDENTERBACKGROUND) {
+		PauseToken pt;
```

Comment:
Same. you're not handling pt anywhere, so why are you introducing it as a new variable?